### DISCUSSION TITLE
import error when run server.py

### DISCUSSION THREAD

**Discussion by BTnoTestUser** [adithya-s-k/omniparse#30] [Q&A]
I followed the Tutorial, but it faild at last step
python server.py --host 0.0.0.0 --port 8000 --documents --media --web

here is my log:
`(omniparse-venv) XXX@XXX:~/onimparse/omniparse$ python server.py --host 0.0.0.0 --port 8000 --documents
Traceback (most recent call last):
  File "/home/XXX/onimparse/omniparse/server.py", line 6, in <module>
    from omniparse import load_omnimodel
  File "/home/XXX/onimparse/omniparse/omniparse/__init__.py", line 27, in <module>
    from marker.models import load_all_models
  File "/home/XXX/miniconda3/envs/omniparse-venv/lib/python3.10/site-packages/marker/__init__.py", line 16, in <module>
    from .settings import __version__
ImportError: cannot import name '__version__' from 'marker.settings' (/home/XXX/miniconda3/envs/omniparse-venv/lib/python3.10/site-packages/marker/settings.py)`

beside some error occur when running
pip install -e .
like
`ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
gradio 4.37.2 requires aiofiles<24.0,>=22.0, but you have aiofiles 0.6.0 which is incompatible.
gradio 4.37.2 requires urllib3~=2.0, but you have urllib3 1.26.19 which is incompatible.`

**Comment by adnan9999** [comment]
how did you resolve it ?
